core/templates/components/main-nav/main-nav.json

```diff
@@ -0,0 +1,84 @@
+{
+  "modifier_class": "",
+  "toggle_text": "Menu",
+  "lv1": [
```

Comment:
Going to have to ponder this data structure a bit. The immediate gap I can see is that a future developer may want to add additional attributes to the link elements and would be unable to do so with this structure.

Reply:
@JBCSU